editor/plugins/gpu_particles_2d_editor_plugin.cpp

```diff
@@ -342,6 +368,7 @@ void GPUParticles2DEditorPlugin::_bind_methods() {
 
 GPUParticles2DEditorPlugin::GPUParticles2DEditorPlugin(EditorNode *p_node) {
 	particles = nullptr;
+	selected_particles.clear();
```

Comment:
It should be empty from the start, so no need to `clear()` in the constructor I think.